scripts/byobu_start.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+# Create new session.
+byobu new-session -d -s Huntsman-Control
```

Comment:
This could be done better, but here's a simple way to parse a first parameter for the name to keep it more generic (I want to use this for regular POCS).

```suggestion
SESSION_NAME=${1-Huntsman-Control)

byobu new-session -d -s "${SESSION_NAME}"
```